src/style.css

```diff
@@ -70,17 +68,17 @@ html {
   height: 4px;
   background-color: var(--blue);
   bottom: 0;
-  border: 50%;
+  border-radius: 8px;
 }
 
 .nav__link:hover {
   color: var(--blue);
 }
 
-li:first-of-type{
+.nav__item:first-child{
```

Comment:
use only margin left of right for nav items so you will avoid margin collapse and won't need to reset margins for first and last children but only for one of them